### PR TITLE
French sources https://chireads.com and https://lnmtlfr.com/

### PR DESCRIPTION
**Pull request by jere344**
* French sources https://chireads.com/ and https://lnmtlfr.com/ 
* English source https://888novel.com/ ( #1349 )
* Fix typos in sources/_sample.py

***

The sources works when using ```--sources [url]``` but when doing a general search they do not show up in the results, I don't know why.

### CODE REVIEW

File: sources/en/0/888novel.py [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+                if self.bad_css:
+                    for bad in div.select(",".join(self.bad_css)):
+                        bad.extract()
+                    # end if
+                # end if
+                for tag in div.find_all(True):
+                    if isinstance(tag, Comment):
+                        tag.extract()  # Remove comments
+
+                    # I want to keep double <br> tags
```
Comment (dipu-bd) [inline sources/en/0/888novel.py:34]: Why do you want to keep <br> tags? In some epub readers it makes the gap between two paragraphs too wide for comfort.

File: sources/fr/lnmtlfr.py [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+            for e in soup.find("div", {"class": "author-content"}).find_all("a")
+        ])
+
+        # Chapter are recuperated from a post request.
+        id = soup.find("input", {"class": "rating-post-id"}).get("value")
+
+        url = "https://lnmtlfr.com/wp-admin/admin-ajax.php"
+        data = f"action=manga_get_chapters&manga={id}"
+        resp = self.submit_form(url, data=data)
+        soup = self.make_soup(resp)
```
Comment (dipu-bd) [inline sources/fr/lnmtlfr.py:59]: Updated this file to avoid using requests and BeautifulSoup directly

File: sources/fr/chireads.py [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+        soup = self.get_soup(self.novel_url)
+        content = soup.find("div", {"class": "conteiner"}).find_all(
+            "div", {"class": "wid"}
+        )
+
+        metadata = content[0]
+        
+        self.novel_cover = self.absolute_url(metadata.find("img").get("src"))
+        
+        self.novel_title = metadata.find("h3", {
```
Comment (dipu-bd) [inline sources/fr/chireads.py:44]: self.cleaner is not necessary. novel title, author, titles in chapters and volumes lists gets formatted later automatically

File: sources/en/0/888novel.py (hunk 1 above)
Comment (jere344) [reply sources/en/0/888novel.py:34]: For me it looks like a single block with one ```<br>``` using sumatrapdf on windows or moon+reader on android. 

double br

single br

Comment (dipu-bd) [reply sources/en/0/888novel.py:34]: ah, I get it. i think i should remove the initialize funtion.

you can use calibre for reading. it has some nice features for formatting ebooks for reading.

**Pull request merged**